.travis.yml

```diff
@@ -1,13 +0,0 @@
-language: ruby
-cache: bundler
-rvm:
-  - 2.7.1
-
-before_install:
-  - gem update --system
-  - npm install -g openapi-enforcer-cli
-
-script:
-  - ./validate-openapi.bash || travis_terminate 1
```

Comment:
Do you think this PR could also remove `validate-openapi.bash`?

Reply:
yes, good catch - just deleted the file